packages/config/src/bin/serve-chrome.ts

```diff
@@ -0,0 +1,164 @@
+
+function checkContainerRuntime(): ContainerRuntime {
+  try {
+    if (execSync('which podman').toString().trim().length > 0) {
+      return 'podman';
+    }
+    if (execSync('which docker').toString().trim().length > 0) {
+      return 'docker';
+    }
+  } catch (error) {
+    throw new Error('No container runtime found');
+  }
+
+  throw new Error('No container runtime found');
+}
+
+async function getLatestCommits(): Promise<string> {
+  const { data } = await axios.get(`https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/commits`, {
+    headers: {
+      Accept: 'application/vnd.github+json',
+      'X-GitHub-Api-Version': '2022-11-28',
+    },
+    params: {
+      per_page: 1,
+    },
+  });
+  if (data.length === 0) {
+    throw new Error('No commits for chrome found!');
+  }
+
+  const { sha } = data[0];
+  return sha.substring(0, 7);
+}
+
+type Deployment = {
+  resourceTemplates: {
+    name: string;
+    targets: {
+      ref: string;
+      namespace: {
+        $ref: string;
+      };
+    }[];
+  }[];
+};
+
+async function getProdRelease() {
+  try {
+    if (!fs.existsSync(chromeDeploymentConfig.tarTarget)) {
+      fs.mkdirSync(chromeDeploymentConfig.tarTarget);
+    }
+    execSync(checkoutCommand, {
+      stdio: 'inherit',
+    });
+
+    const deployment = load(
+      fs.readFileSync(path.join(chromeDeploymentConfig.tarTarget, chromeDeploymentConfig.deployFile), {
+        encoding: 'utf-8',
+      })
+    ) as Deployment;
+    const chromeProd = deployment.resourceTemplates
+      .find((template) => {
+        return template.name === 'insights-chrome';
+      })
+      ?.targets.find((target) => {
+        return target.namespace.$ref.includes('prod-frontends');
+      });
+    if (!chromeProd) {
+      throw new Error('Unable to find chrome prod deployment configuration.');
+    }
+
+    return chromeProd.ref.substring(0, 7);
+  } catch (error) {
+    fecLogger(LogType.error, error);
+    fecLogger(LogType.warn, 'Unable to find chrome prod deployment! Falling back to latest image.');
+    return getLatestCommits();
+  }
+
+  // const { data: deploymentYaml } = await axios.get(deploymentSource);
+  // console.log(deploymentYaml);
+}
+
+function pullImage(tag: string) {
+  execSync(`${execBin} pull quay.io/cloudservices/insights-chrome-frontend:${tag}`, {
+    stdio: 'inherit',
+  });
+}
+
+function startServer(tag: string) {
+  try {
+    execSync(`${execBin} stop fec-chrome-local`, {
+      stdio: 'inherit',
+    });
+    execSync(`${execBin} rm fec-chrome-local`, {
+      stdio: 'inherit',
+    });
+  } catch (error) {
+    fecLogger(LogType.info, 'No existing chrome container found');
+  }
+  const child = spawn(
+    `${execBin} run -p ${EXPOSED_PORT}:${CONTAINER_PORT} --name fec-chrome-local quay.io/cloudservices/insights-chrome-frontend:${tag}`,
```

Comment:
Why this runs containers from node? Have you considered a simpler approach with podman-compose?

Reply:
We are trying to keep the docker/podman dependencies to a bare minimum. We also don't want to change the developer workflow at all. One should still just have to hit the `npm run start` and the rest happens behind the scenes.

Reply:
Sorry Martin, but this re-implements what the compose already does the best. Backend dev already utilize podman/docker-compose, so this would not be something new. `npm run start` can initiate compose command to keep frontend workflow the same.

Reply:
> Sorry Martin, but this re-implements what the compose already does the best. 

I see. I was under the impression you were suggesting switching from `npm` script to docker/podman compose.

> Backend dev already utilize podman/docker-compose, so this would not be something new.

This utility is not for backend devs :) I will defer to my original point. We want to keep the requirements to a minimum. I am fully aware and expect that you will counter my opinion by saying compose (docker or podman) is something standard that every developer should have/know.

But we do not benefit in any way from running compose instead of a simple node. We do not allow any configuration and it is not needed. The only thing we need is a simple static web server. With the exception of a port to run the web server (in case the default is occupied), we do not need or in fact, do not want anyone to change anything. Because there is nothing to change within the frontend containers. There are no dependencies, volumes, extra exposed ports, nothing. We are talking about an asset server, not rest API, or DB, or message bus, that would require a bit more control. If Chrome is supposed to be pointed to a local repository, there is already a way to do so without containers. The only thing that compose would do right now, is to introduce a new requirement to install it. With no benefit to the developer because they can't even configure it.

Reply:
My suggestion is to make it simple and maintainable. What may seem simple does not necessarily mean that it would be simple to maintain. (Less is more :slightly_smiling_face:) 

Since this introduces containers to serve static content, then a convenient method of running bunch of containers would be to use compose. Indeed, it does add a dependency. If we, however, want to keep it simple, we do not need containers to serve static assets (Node can do it). I think that our obstacle is that the static content is provided through container images (instead of archives). To keep this simplish, this can extract the assets from the image and then serve it from a temporary directory. No container runtime is required.

Reply:
> To keep this simplish, this can extract the assets from the image and then serve it from a temporary directory. No container runtime is required.

AFAIK you still need the runtime to extract the assets themselves. YOu'd have to start it, extract the assets, stop the container, and start a node server. Don't think that is simpler. If there is a way to extract files from a container/image without using runtime, I am interested in learning.

I'll re-iterate. If we ever move towards serving locally built assets from docker rather than using the webpack dev server, using compose will be the obvious pick. But at this point, I don't see any benefit. If we decide to switch from webpack dev server, rewriting a docker node to compose, will be the least of our problems.

Reply:
There is a `save` command that gets an image as tar. However, the inners of an image tar are image layers that need to be unpacked.

```
# extract to current dir
podman save ImagID| tar xf -
```

This might be a bit clumsy.

Reply:
That still requires the runtime installed though. Why not just run it when you have to download it anyway?